### PR TITLE
Added functionality to handle json test cases.

### PR DESCRIPTION
**Pull request by punjabisingh**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Tested using json formated test case. These test cases are same for all the utilities and making sure all test cases pases.

### CODE REVIEW

**Comment by robincher** [comment]
@punjabisingh  Please fix the quality issues and failing builds first.

The CI is throwing this error 

```
/usr/lib/mono/msbuild/15.0/bin/Microsoft.Common.CurrentVersion.targets(2067,5): warning MSB3245: Could not resolve this reference. Could not locate the assembly "BouncyCastle.Crypto". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors. [/home/travis/build/GovTechSG/csharp-apex-api-security/ApiUtilLib/ApiUtilLib.csproj]```

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,21 @@
 # Change Log
 
 ## Added 
+<<<<<<< HEAD
```
Comment (robincher) [inline CHANGELOG.md]: Can resolve the conflicts and update the new version for change log?

**Review by robincher — CHANGES_REQUESTED** [review]
General feedback
1) Fix avoidable quality issues mentioned by codacy
2) Resolve the conflicts of CHANGELOG.md
3) Is it necessary to upload the binary files inside .vs/ ? If not, please gitignore them.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,12 @@
 # Change Log
 
 ## Added 
-### V1.0-SNAPSHOT
-+ 2017-10-29 - Initial release.
+
+### V1.1-SNAPSHOT
++ 2018-09-10 - Bug Fixes.
```
Comment (robincher) [inline CHANGELOG.md:6]: Duplicate version?

File: ApiUtilLibTest/CommonExtensions.cs [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                value = value.Replace(item, "");
+            }
+
+            return value;
+        }
+
+        public static bool ToBool(this string value)
+        {
+            if(!value.IsNullOrEmpty()){
+                if (value.ToLower() == "true")
```
[the remark this answers is not in the document]
Comment (punjabisingh) [reply ApiUtilLibTest/CommonExtensions.cs:114]: this is fine one liner code

**Review by robincher — APPROVED** [review]

**Pull request merged**